pkg/deploy/generator/resources_dev.go

```diff
@@ -316,6 +316,36 @@ for attempt in {1..5}; do
   if [[ ${attempt} -lt 5 ]]; then sleep 10; else exit 1; fi
 done
 
+DEVICE_PARTITION=$(pvs | grep '/dev/' | awk '{print $1}' | grep -oP '[a-z]{3}[0-9]$')
+DEVICE=$(echo $DEVICE_PARTITION | grep -oP '^[a-z]{3}')
+
+# Fix the "GPT PMBR size mismatch (134217727 != 268435455)"
+echo "w" | fdisk /dev/${DEVICE}
+
+# Steps from https://access.redhat.com/solutions/5808001
+# 1. Delete the LVM partition
+# 2. Recreate the partition
+# 3. Accept the default start and end sectors (2 x \n)
+# 4. DO NOT delete the LVM2_member signature
```

Comment:
either the doc is out of date or something is a little off here. After `n` the solution you linked outlines:

a. selecting the partition type of Primary (default) vs Extended
b. selecting the partition number (default)
c. setting the first sector (default)
d. setting the last sector (default)
e. a `y/n` for removing the signature (no default)

This comment calls for 2 `\n`, but there are 3 in the script, and the doc calls for 4 + a `y/n` response that is commented but not in the script. Can you clarify what's going on here?

Reply:
It might add clarity to put the actual commands as part of these comments:

```
# 1. Delete the LVM partition "d\n2\n"
# 2. Recreate the partition as a primary with default partition number, default start sector (first available), default end sector (last available), and without removing the signature "n\np\n2\n\n\nn\n"
# 3. Set the type of the partition "t\n2\n31\n"
# 4. Write the table "w\n"
```

Reply:
I see what you mean.  The solution shows you select partition type but I am never prompted for that locally.  I'm not certain why it is this way.  I will also state that it doesn't look like `@darthhexx` encountered this when running it ad hoc.

As for the comment misaligned with the script, I'll push a update to clarify that.